lib/tests/simple_confs/a.conf

```diff
@@ -0,0 +1,11 @@
+{
+    include file("d.conf")
+    Peter : {
+        include file("b.conf")
+        include file("sub/b.conf")
+        # passwd4 : ${new_passwd}
```

Comment:
This didn't work?

Reply:
This works。I don't remember what I was testing that I put a sharp in front of it.